analysers/analyser_osmosis_powerline.py

```diff
@@ -29,20 +29,67 @@
 sql01 = """
 CREATE TEMP TABLE power_lines_nodes AS
 SELECT
-    ways.id as wid,
-    unnest(ways.nodes) AS nid,
-    unnest(ways.nodes[2:array_length(ways.nodes,1)]) AS nid_next,
-    ways.tags->'cables' as cables,
-    ways.tags->'circuits' as circuits,
-    coalesce(ways.tags->'location','overhead') as location,
+    w.id as wid,
+    unnest('{NULL}'||w.nodes[1:array_length(w.nodes,1)-1]) AS nid_prec,
+    unnest(w.nodes) AS nid,
+    unnest(w.nodes[2:array_length(w.nodes,1)]) AS nid_next,
+    w.tags->'cables' as cables,
+    w.tags->'circuits' as circuits,
+    coalesce(w.tags->'location','overhead') as location,
     voltage
 FROM
-    ways
-    JOIN LATERAL (SELECT array_agg(lpad(v, 99, '0')) FROM unnest(regexp_split_to_array(ways.tags->'voltage','; *')) AS t(v)) AS t(voltage) ON TRUE
+    ways w
+    JOIN LATERAL (SELECT array_agg(lpad(v, 99, '0')) FROM unnest(regexp_split_to_array(w.tags->'voltage','; *')) AS t(v)) AS t(voltage) ON TRUE
```

Comment:
@frodrigo do you remember why this 99 zeros padding is needed for voltages?
According to current code, I'd be in favour of removing it

Reply:
I guess it is a trick to compare with non number values.

Reply:
Indeed, so I shouldn't remove it and improve the handling of non number values